src/Core/Logging/Extensions/ILogServiceExtensions.Info.cs

```diff
@@ -0,0 +1,166 @@
+﻿using System;
+using System.Runtime.CompilerServices;
+
+namespace CodeMonkeys.Logging
+{
+    public static partial class LogServiceExtensions
+    {
+        public static void Info<TState>(
+            this ILogService service,
+            DateTimeOffset timestamp,
+            TState state,
+            Exception ex,
+            [CallerMemberName] string methodName = "")
+        {
+            service.Internal(
+                timestamp,
+                state,
+                ex,
+                methodName);
+        }
+
+        public static void Info<TState>(
+            this ILogService service,
+            DateTimeOffset timestamp,
+            TState state,
+            [CallerMemberName] string methodName = "")
+        {
+            service.Internal(
+                timestamp,
+                state,
+                methodName);
+        }
+
+        public static void Info(
+            this ILogService service,
+            DateTimeOffset timestamp,
+            Exception ex,
+            [CallerMemberName] string methodName = "")
+        {
+            service.Internal(
+                timestamp,
+                ex,
+                methodName);
+        }
+
+
+
+        public static void Info<TState>(
+            this ILogService service,
+            TState message,
+            Exception ex,
+            [CallerMemberName] string methodName = "")
+        {
+            service.Internal(
+                message,
+                ex,
+                methodName);
+        }
+
+        public static void Info<TState>(
+            this ILogService service,
+            TState message,
+            [CallerMemberName] string methodName = "")
+        {
+            service.Internal(
```

Comment:
Codacy found an issue: [Refactor this method to add validation of parameter 'service' before using it.](https://app.codacy.com/gh/UltimateCodeMonkeys/CodeMonkeysMVVM/pullRequest?prid=5762132)